core-web/libs/edit-content/src/lib/fields/dot-edit-binary-field/dot-edit-binary-field.component.ts

```diff
@@ -42,11 +42,10 @@ import { DotBinaryFieldUrlModeComponent } from './components/dot-binary-field-ur
 import { BinaryFieldMode, BinaryFieldStatus } from './interfaces';
 import { DotBinaryFieldEditImageService } from './service/dot-binary-field-edit-image/dot-binary-field-edit-image.service';
 import { DotBinaryFieldStore } from './store/binary-field.store';
-
-import { getUiMessage } from '../../utils/binary-field-utils';
+import { getUiMessage } from './utils/binary-field-utils';
 
 @Component({
-    selector: 'dot-binary-field',
+    selector: 'dot-edit-binary-field',
```

Comment:
We are using `dot-edit-content-xxx` as standard, should we do that for this component too?

Reply:
@fmontes Do we want to rename the child components as well? Meaning `dot-binary-field-editor` to `dot-edit-content-binary-field-editor`, and so on?